cardano-api/src/Cardano/Api/Key.hs

```diff
@@ -67,6 +69,16 @@ generateSigningKey keytype = do
     seedSize = deterministicSigningKeySeedSize keytype
 
 
+generateInsecureSigningKey :: (Key keyrole, SerialiseAsRawBytes (SigningKey keyrole))
+  => AsType keyrole
+  -> IO (SigningKey keyrole)
+generateInsecureSigningKey keytype = do
+  g <- Random.getStdGen
+  let (bs, _) = Random.genByteString (fromIntegral (deterministicSigningKeySeedSize keytype)) g
```

Comment:
Can we ditch a pair of the parentheses with `$`?

Reply:
Done